client/reader/controller.js

```diff
@@ -19,7 +19,8 @@ var i18n = require( 'lib/mixins/i18n' ),
 	i18n = require( 'lib/mixins/i18n' ),
 	TitleStore = require( 'lib/screen-title/store' ),
 	titleActions = require( 'lib/screen-title/actions' ),
-	FeedSubscriptionActions = require( 'lib/reader-feed-subscriptions/actions' );
+	FeedSubscriptionActions = require( 'lib/reader-feed-subscriptions/actions' ),
+	urlHelper = require( 'reader/url-helper' );
```

Comment:
I like the idea moving it to `lib/route/reader.js`. @bluefuton or @blowery any thoughts?